### PR TITLE
add ability to add shows within django app. Tests included

### LINKED ISSUE #17: Ability to Add Shows
Authenticated users can add shows with associated artist and venues (eventually we'll want only admins, but we'll add that after we create users with different privileges)

### PR DESCRIPTION
**Pull request by haselfine**
@jy8474np @whagestad @JHughes9802 @quinzel25 @alicia-mctc 

Added ability to add shows from the django app itself. You must be logged in.

I had a little trouble getting the tests to run. I wrote one in test_models. If anyone could help figure out what's going wrong there, that'd be awesome and then I could write more tests.

### CODE REVIEW

File: lmn/tests/test_models.py [hunk 1]
```diff
@@ -26,3 +33,21 @@ def test_create_user_duplicate_email_fails(self):
...
 
 
+class TestShow(TestCase):
+
+    def setUp(self):
+        self.user = User.objects.get(pk=1)
+        self.client.force_login(self.user)
+        
+    def test_create_new_show(self):
+        fixtures = ['testing_artists', 'testing_venues']
```
Comment (claraj) [inline lmn/tests/test_models.py]: List the fixtures outside of this method, part of the TestShow class

**Comment by quinzel25** [comment]
Adding a show works well and shows up in the database. I also ran the tests and all seemed to pass besides one in test_views.py which I believe you didn't modify

Screenshot 2020-11-23 155845

**Comment by claraj** [comment]
Was that test passing when you got the code? If so then investigate why it's failing now

**Review by alicia-mctc — COMMENTED** [review]
I could not figure out how to make the test_models.py to work either. Please add comments.

**Comment by haselfine** [comment]
Passing tests have successfully been added. Located at bottom of test_views.py. Comments added to views_shows and tests.

**Comment by whagestad** [comment]
Add show functionality is working on my end. I have an issue with my configuration, I expect it's an issue with selenium. Will get that fixed and report back if I have any issues running your tests. Thanks Ben.

**Review by whagestad — APPROVED** [review]

**Review by alicia-mctc — APPROVED** [review]
Great job on unit testing.

**Pull request merged**